calc_multiply.py

```diff
@@ -0,0 +1,13 @@
+def mult():
+    a,b = None
```

Comment:
a,b = None produces an error for me when I run it, but is fine when it is removed. I might be missing something, let me know.